frontend/src/Settings/Notifications/Notifications/Notification.js

```diff
@@ -80,55 +84,78 @@ class Notification extends Component {
         </div>
 
         {
-          supportsOnGrab && onGrab &&
+          supportsOnGrab && onGrab ?
             <Label kind={kinds.SUCCESS}>
-              {translate('OnGrab')}
-            </Label>
+              {translate('OnGrab')};
```

Comment:
Get rid of the `;` it comes through literal in js to the label

![image](https://user-images.githubusercontent.com/376117/110052521-b3477a00-7d25-11eb-89f5-60dd590cfa53.png)

Reply:
Looking at that image, OnUpgrade is not translated or something is wrong with the key it seems. Or the translation has no space

Reply:
Yeah the translation key was missing! Added it